common/static/coffee/spec/discussion/view/discussion_view_spec_helper.coffee

```diff
@@ -23,6 +23,14 @@ class @DiscussionViewSpecHelper
         }
         $.extend(thread, props)
 
+    @checkVoteClasses = (view) ->
```

Comment:
@attiyaIshaque params mismatch? We are calling `checkVoteClasses` with two params. 
`DiscussionViewSpecHelper.checkVoteClasses(@view, @thread)`